src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
+
+    private static final String OPERATION_TYPE_BUY = "buy";
+    private static final String OPERATION_TYPE_SUPPLY = "supply";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        List<String> stringFromFile = new ArrayList<>();
+        StringBuilder report = new StringBuilder();
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String read = bufferedReader.readLine();
+            while (read != null) {
+                stringFromFile.add(read);
+                read = bufferedReader.readLine();
+            }
+            int buy = 0;
+            int supply = 0;
+
+            for (String s : stringFromFile) {
+                String[] split = s.split(",");
+                int digit = Integer.decode(split[1]);
```

Comment:
Use `Integer.parseInt` instead of `Integer.decode` here. `Integer.decode` is used for decoding strings into integers, but since the string is already in decimal format, `parseInt` is more appropriate.